docs/ui/components/BoxLink/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { css } from '@emotion/react';
+import { borderRadius, spacing, theme, ArrowRightIcon, iconSize, shadows } from '@expo/styleguide';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { A, HEADLINE, P } from '~/ui/components/Text';
+
+type BoxLinkProps = PropsWithChildren<{
+  title: string;
+  description: string | ReactNode;
+  href?: string;
+  testID?: string;
+}>;
+
+export function BoxLink({ title, description, href, testID }: BoxLinkProps) {
+  return (
+    <A href={href} css={tileContainerStyle} data-testid={testID}>
+      <HEADLINE tag="span">{title}</HEADLINE>
+      <P>{description}</P>
+      <ArrowRightIcon css={iconStyle} color={theme.icon.secondary} size={iconSize.large} />
```

Comment:
```suggestion
      <ArrowRightIcon css={iconStyle} color={theme.icon.secondary} />
```
This is optional: but I think making these smaller will look nicer.

Reply:
I have tested both sizes while implementing component, both looks good, we can stick to the regular one.